server/owlEye/app/index.py

```diff
@@ -23,8 +14,10 @@
             'Access-Control-Allow-Origin': '*',
             'Access-Control-Allow-Methods': 'OPTIONS,POST,GET'
         }
+TABLE_NAME =  os.getenv("JOB_TABLE")
```

Comment:
I believe accessing environment variables with os.getenv() and os.environ.get() when it doesnt exist will return None. I believe it is better practice to use os.environ["JOB_TABLE"] instead, as this is a non-optional environment variable an you wish to raise an KeyError if it doesn't exist